recipes/a5-miseq/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set name = "A5-miseq" %}
+{% set version = "20160825" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name|lower }}-{{ version }}.tar.gz
+  url: https://sourceforge.net/projects/ngopt/files/a5_miseq_linux_20160825.tar.gz/download
```

Comment:
Can you remove the /download from the URL

Reply:
Done.